lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -192,7 +192,12 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
     end)
     |> Enum.flat_map(fn
       route_id ->
-        headsign = get_destination(t, location, route_id)
+        # Boundary alerts shouldn't have headsign in the banner
+        headsign = if get_region_from_location(location) === :boundary do
+            nil
+          else
+            get_destination(t, location, route_id)
+          end
```

Comment:
Could condense this a little with `unless`:
```suggestion
        headsign = unless get_region_from_location(location) === :boundary do
            get_destination(t, location, route_id)
          end
```